lib/DDG/Spice/IndeedJobs.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::IndeedJobs;
+# ABSTRACT: Search for jobs on Indeed.
+# Reviews DDG_SPICE_INDEED_JOBS_PUBLISHER environment variable
+# https://ads.indeed.com/jobroll/xmlfeed
+
+use DDG::Spice;
+
+primary_example_queries "java jobs";
+secondary_example_queries "perl jobs in san francisco";
+description "Indeed jobs";
+name "Indeed Jobs";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/IndeedJobs.pm";
+topics "special_interest";
+category  "reference";
+attribution github  => ['parker', 'Parker'],
+            twitter => ['pseidel', 'Parker'],
+            github  => ['tagawa', 'Daniel Davis'],
+            twitter => ['ourmaninjapan', 'Daniel Davis'],
+            web     => ['http://daniemon.com/', 'Daniel Davis'];
+
+triggers any => "job", "jobs";
+
+spice to => 'http://api.indeed.com/ads/apisearch?publisher={{ENV{DDG_SPICE_INDEED_APIKEY}}}&v=2&useragent=DuckDuckGo&userip=1.2.3.4&q=$1&l=$2&co=$3&format=json&callback={{callback}}';
+spice from => '([^/]+)/(.*?)/([^/]*)';
+spice proxy_cache_valid   => "418 1d";
+
+handle query => sub {
+    my $country = 'US';
+    if($loc->country_code) {
+        $country = $loc->country_code;
+    }
+    my $spice_location = $loc->city . ', ' . $loc->region_name;
+    if (/(?:\s*(?:i\s+|we\s+)?(?:need|want|wanna|deserve|seek|get|find)\s+(?:a\s+|an?\s+)?)?(?:(.+)\s+)?(?:jobs?|work|employment|internship)(?:\s+(?:in|near|around\s+)?\s*(.+))?$/i) {
+        if ($1 || $2) {
+            my $query = $1 || ' ';
+            my $location = $2 || $spice_location || ' ';
+            return $query, $location, $country;
```

Comment:
Still wondering about the best way to handle countries. You're right that they support more than just the US but they can't handle the whole location without country being a separate parameter.

I think most practical would be a hash lookup of capitals/countries, and perhaps a separate check for the country name in the query. This still wouldn't catch your Philadelphia (Canada) example but it would be better than nothing.